stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.11.4
+version: 1.11.5
```

Comment:
We should do a minor version bump. `1.12.0`

Reply:
chnaged